aio/content/marketing/contributors.json

```
@@ -306,7 +247,7 @@
     "twitter": "manekinekko",
     "website": "https://medium.com/@wassimchegham",
     "bio": "Wassim (aka manekinekko on Twitter/Github) is a Developer Advocate at SFEIR, in Web technologies (Angular, Polymer, PWA, Web Components...). He is also a Developer Expert in Web technologies nominated by Google. He enjoys writing technical articles, meeting developers at events, speaking at conferences and contributing to open source projects. Wassim loves the Web Platform and works hard to move it forward.",
-    "groups": ["Angular"],
+    "groups": ["Collaborators"],
```

I have no idea if this is correct, but since @StephenFluin approved, I guess it is.
Wassim is also a GDE, though, so `groups` should be:

```suggestion
    "groups": ["Collaborators", "GDE"],
```